account/symbol_multisig_relink.py

```diff
@@ -39,13 +40,16 @@ def _prepare_aggregate_transfer(self, key_pair_repository, transaction_dict):
                 })
 
         aggregate_transaction = aggregate_builder.build(transaction_dict['fee_multiplier'], {'deadline': deadline})
-        aggregate_transaction.transactions[0].mosaics[0] = (transaction_dict['fee_mosaic_id'], aggregate_transaction.fee)
+        currency_mosaic = aggregate_transaction.transactions[0].mosaics[0]
```

Comment:
why assignment here, you're overwriting all fields aren't you?

Reply:
yea, not sure what I was thinking 😆